tests/fib/test_fib.py

```diff
@@ -78,6 +78,7 @@ def test_basic_fib(duthosts, ptfhost, ipv4, ipv6, mtu,
         wait(30, 'Wait some time for mux active/standby state to be stable after toggled mux state')
 
     timestamp = datetime.now().strftime('%Y-%m-%d-%H:%M:%S')
+    switch_type = duthosts[0].facts['switch_type']
```

Comment:
@arlakshm : i think even for pizza box we should use new algorithm as this is correct way for multi-hop routing.

Reply:
@arlakshm , On a single asic pizza box, this change will not affect the outcome as the expected count will be the same as before. The difference kicks in when we have multi-asic, multi-lc kind of system, there in how the routing decision is made becomes differentiating factor. @abdosi, what happens if we have multi-asic pizza box, there in this change may affect the expected count calculation. For now, if you want me to make the change only for packet-chassis, I can do that. Kindly advise since you have a better view on all different kind of switches.